docs/release-notes.md

```diff
@@ -1,47 +1,45 @@
+
 
 #### [MPS](Glossary.md#m)
-- **CORS Support:** Added support for Cross-Origin Resource Sharing (CORS).
+- **PostgreSQL integration added to MPS:** To support additional metadata we are adding to MPS, we have added a PostgreSQL reference implementation to MPS.
+- **Hostname and Device Metadata tags:** MPS will store device hostname and metadata tag information it receives from RPS in it's PostgreSQL DB.  When device information calls are made, the device hostname will be returned to the caller along with the other device information previously provided.  Additionally, the GET devices call now supports an additional tags field (comma-delimited list) that will filter the devices based on the set of tags provided.  This greatly improves scaling of this call with very large number of devices as well as reduces the need for filtering on the caller's side.
+- **Node 14 LTS support:** RPS has been updated to support running on Node 14 LTS. 
+- **MPS Scaling *preview* feature:** We've added a scaling configuration feature to MPS that will allow MPS to scale up multiple instances of MPS in a Kubernetes environment to support large deployments.  Both the CIRA connection side and the REST API side of MPS can be scaled up independently.  When an MPS instance goes down, Intel&reg; AMT device connections will be automatically reconnected to a different instance of MPS and the route to that device will be automatically updated.  The current implementation is designed to run in an Azure Kubernetes Service behind a load balancer.  As scaling is a preview feature, it currently doesn't support socket based Intel&reg; AMT connections (KVM and SOL) or the new Metadata (hostnames and tags) feature.  Other features such as PowerAction, HardwareInfo, EventLog, etc should all function in scaling mode.  We expect to have all AMT features supported in scaling mode in the 1.3 release.
 
-#### [UI Toolkit](Glossary.md#u)
-- **CORS support:** Added support for Cross-Origin Resource Sharing (CORS).
-- **MEBx Password option added to Profile Editor:** Provides the ability to set a MEBx password when activating Intel&reg; AMT devices.
-- **Network Setting option added to Profile Editor:** Provides the ability to set either DHCP or Static IP setting when activating Intel&reg; AMT devices.
-- **Edit feature added to Profile Editor:** Added ability to edit AMT, CIRA, and Domain profiles in the Profile Editor.
+#### [Sample UI](Glossary.md#s)
+- **New look & feel** The Sample UI has had a full rewrite in the 1.2 release based on much feedback from our customers.  This updated UI supports the new hostname and metadata tag features.  While this is still just a reference implementation to showcase the features of the MPS and RPS components, this restructuring of the Sample UI will:
+    * Aid developers as an example of how to use our APIs 
+    * Help understand the value of Open AMT Cloud Toolkit.
```

Comment:
I actually wrote something about this in one my wireframe examples.

Use the Sample UI in the pre-deployment phase of solution development to:
-learn how the Open AMT Cloud Toolkit connects development systems to managed devices.
-set up profiles, configurations, and domains.
-understand Open AMT's implementation of the REST APIs and how they can be used in your console application. 

You can use it or not. It's up to you.